STYLE-GUIDE.md

```diff
@@ -1,2 +1,77 @@
 ### Refer to Airbnb JavaScript Style Guide
-  - https://github.com/airbnb/javascript
\ No newline at end of file
+  - https://github.com/airbnb/javascript
+
+
+### Writing the React Component
+Prefer to use class-based components instead of pure function,
+because the class-based components can do much more than pure function,
```

Comment:
Rewrite as "because the former can..."
Don't just say "do much more than pure function", which is less informative.